app/main.py

```diff
@@ -1,13 +1,31 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line":
+            error["line_number"],
+        "column":
+            error["column_number"],
+        "message":
+            error["text"],
+        "name":
+            error["code"],
+        "source":
+            "flake8"
+    }
 
 
-def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+def format_single_linter_file(file_path: str, errors: dict) -> dict:
+    return {
+        "errors":
+            [
+                format_linter_error(error) for error in errors
+            ],
+        "path": file_path,
+        "status": "failed" if len(errors) > 0 else "passed"
+    }
 
 
-def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+def format_linter_report(errors_linter: dict) -> list:
+    return [
+        format_single_linter_file(key, value) for key,
+        value in errors_linter.items()
```

Comment:
```suggestion
        format_single_linter_file(key, value) 
        for key, value in errors_linter.items()
```
Change formatting. Also, key and value is not a good variable name